src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/MainErrorReportViewControllers/BottomErrorReportViewController.xib

```diff
@@ -139,5 +167,8 @@
         <namedColor name="ENA Separator Color">
             <color red="0.96078431372549022" green="0.96078431372549022" blue="0.96078431372549022" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
         </namedColor>
+        <namedColor name="ENA Text Primary 2 Color">
+            <color red="0.090196078431372548" green="0.098039215686274508" blue="0.10196078431372549" alpha="0.60000002384185791" colorSpace="custom" customColorSpace="sRGB"/>
+        </namedColor>
```

Comment:
Just asking: Have we ever got a definition for these colors?

Reply:
No and i couldn't get it from figma, but i saw it was the same color used in the history cells implemented by naveed so i just used it.